transforms/create-element-to-jsx.js

```diff
@@ -66,11 +66,23 @@ module.exports = function(file, api, options) {
     }
   };
 
+  const jsxIdentifierFor = node => {
```

Comment:
sounds good! I honestly didn't know that jsxMemberExpression was something different from a MemberExpression, that's silly.